cartridges/int_externalCustomerData/cartridge/int_externalCustomerData.properties

```diff
@@ -0,0 +1,4 @@
+## cartridge.properties for cartridge int_externalCustomerData
+#Thu Oct 9 14:2:14 CEST 2023
+demandware.cartridges.int_externalCustomerData.multipleLanguageStorefront=true
+demandware.cartridges.int_externalCustomerData.id=int_externalCustomerData
```

Comment:
cartridge name should be lower case only

Reply:
fixed